StepmaniaCore.cmake

```diff
@@ -231,6 +231,9 @@ find_package(Iconv)
 find_package(Threads)
 if (${Threads_FOUND})
   set(HAS_PTHREAD TRUE)
+  list(APPEND CMAKE_REQUIRED_LIBRARIES pthread)
+  check_symbol_exists(pthread_mutex_timedlock pthread.h HAVE_PTHREAD_MUTEX_TIMEDLOCK)
```

Comment:
...admittedly not the most standard of places to put this, but I understand where you are coming from.